notebooks/community/ml_ops/stage2/get_started_vertex_tensorboard.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Need to add Open in Colab</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>